web/src/main/java/org/cbioportal/web/parameter/StudyPageSettings.java

```diff
@@ -0,0 +1,212 @@
+package org.cbioportal.web.parameter;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import javax.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+/**
+ * @author kalletlak
+ *
+ */
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+public class StudyPageSettings extends SessionData implements Serializable {
```

Comment:
Even for these settings. Do we have to model them?

Reply:
this would be helpful to catch errors